internal/espresso/config.go

```diff
@@ -116,9 +116,6 @@ func Validate(p Project) error {
 		if err := validateEmulators(suite.Name, suite.Emulators); err != nil {
 			return err
 		}
-		if suite.TestOptions.NumShards != nil && suite.TestOptions.ShardIndex == nil {
-			return fmt.Errorf("missing shardIndex")
```

Comment:
I think a warning here is a good idea. If `numShards` is defined without a `shardIndex`, that could imply the customer wants sharding but has just misconfigured his saucectl run. Some output to indicate a possible misconfiguration could be helpful.